src/Microsoft.AspNetCore.Mvc.ViewFeatures/MvcViewOptions.cs

```diff
@@ -33,6 +46,36 @@ public HtmlHelperOptions HtmlHelperOptions
             }
         }
 
+        /// <summary>
+        /// Gets or sets a value that determines if the <see cref="ITempDataDictionary"/> keys for
+        /// properties annotated with <see cref="TempDataAttribute"/> include the prefix <c>TempDataProperty-</c>.
+        /// Defaults to <c>false</c>.
+        /// </summary>
+        /// <remarks>
```

Comment:
Can you add another paragraph here that clearly explains what it means? We don't want to pack 100% of everything in the summary, but the summary is not enough.